src/pages/Portfolio/ImpactDashboard/ImpactDashboardPage.vue

```diff
@@ -59,13 +60,31 @@ export default {
 	},
 	data() {
 		return {
-			showEdModule: true
+			post: null
 		};
 	},
 	methods: {
-		hideModule(payload) {
-			this.showEdModule = !payload;
+		loadEducationPost() {
+			// Donation Education Module Experiment MARS-497
+			this.apollo.query({
+				query: gql`query ContentfulBlogPosts (
+						$customFields: String,
+						$limit: Int
+					) {
+						contentful {
+							blogPosts: entries(contentType:"blogPost", customFields:$customFields, limit:$limit)
+						}
+					}`,
+				variables: {
+					customFields: 'metadata.tags.sys.id[in]=impact-page|order=-fields.originalPublishDate'
+				},
+			}).then(({ data }) => {
+				this.post = data?.contentful?.blogPosts?.items?.[0]?.fields ?? null;
+			});
 		}
+	},
+	created() {
```

Comment:
Let's use `mounted` instead so that this isn't started on the server-side. The server completes the request as soon as `created` finishes, so anything async started during that is ignored.